connectors/package.json

```diff
@@ -29,7 +32,7 @@
   },
   "devDependencies": {
     "@types/eslint": "^8.21.3",
-    "@types/node": "^18.15.5",
+    "@types/node": "^18.16.0",
```

Comment:
we should align on `front` and use nvm ?

Reply:
It does use NVM, not sure why I updated the types 😬  removing

Reply:
fixed